### DISCUSSION TITLE
how to set gpt temperature if using azure openai for metrics?

### DISCUSSION THREAD

**Discussion by hbghub** [confident-ai/deepeval#1150] [Q&A]
I used CLI "deepeval set-azure-openai" to config deepeval to use Azure OpenAI for metrics computation. However, I don't see an option for "temperature".

Would deepeval set "temperature" to some default value?

When I use deepeval to compute faithfulness for example, I noticed the value could vary a lot from one run to the other, even if the context and outputs are the same. So I wonder if that is caused due to the "temperature" of openai not setting up 0.0?

Thanks for the help!

**Comment by penguine-ip** [comment]
Hey @hbghub right now there's no option in both CLI and code but we'll add it to our integrations in the next release. We hope that you can provide a kwarg in code: https://www.deepeval.com/integrations/models/azure-openai#python
